src/app/components/Inside/Sidebar/Sidebar.jsx

```diff
@@ -84,17 +88,17 @@ class Sidebar extends React.Component {
           <div className={classes.footer}>
             <span>
               <a href={'https://github.com/Musish/Musish/issues/new/choose'} target={'_blank'}>
-                Feedback
+                {translate.feedback}
               </a>
               {' & '}
               <a href={'https://github.com/Musish/Musish'} target={'_blank'}>
                 Github
               </a>
             </span>
             <span className={classes.footnote}>
-              {'Designed with '}
+              {translate.feedbackMessageOne}
```

Comment:
This should all be one localised string, otherwise we can't control where the heart icon appears in the sentence.